test/cache/repair/reinstalls_git_packages_test.dart

```diff
@@ -28,7 +28,7 @@ main() {
     pubUpgrade();
 
     // Break them.
-    var fooDirs;
+    List fooDirs;
```

Comment:
You're welcome to make changes in tests, but be aware that there's no infrastructure in place to make sure they stay strong-mode clean.